### DISCUSSION TITLE
Improving the project name

### DISCUSSION THREAD

**Discussion by Hebgbs** [FossifyOrg/General-Discussion#30] [General]
Fossify? Really? Coming fresh from Brodie's video about the project, there are some other, more _clever_ names you could had used. (BTW appreciate the effort, just a little gentle ribbing and stoking the fires of passion to hopefully give this project a more punchy name.)

PureDroid / Pure Tools were already discussed elsewhere, why not CleanDroid or Clean Tools? The collective project as CleanSuite? Or maybe, even better considering the moral of SMT's story was a fable of commercialisation, and that this name also contains all letters for the Android Open Source Project (AOSP), **AesopTools.**

Throw your hat in the ring and let's see what sticks.

**Comment by Aga-C** [comment]
It's already explained that the name won't be changed: https://github.com/FossifyX/General-Discussion/issues/2#issuecomment-1839364055.

And if you'd like to discuss certain app names, there is an ongoing poll: https://github.com/orgs/FossifyX/discussions/23.

**Comment by naveensingh** [comment]
@Hebgbs Well, naming is hard.

If we name it Clean Tools, who's to say it won't be associated with [this](https://www.google.com/search?q=why+clean+master+is+removed+from+play+store)?

Also, CleanDroid: https://www.google.com/search?q=CleanDroid

Meanwhile: https://www.google.com/search?q=Fossify (check the first link)

I understand that the name is not interesting to the general population but I believe people will get used to it. When I first learned the name "Google", it sounded weird to me and I was 14!

And respectfully, Brodie Robertson was part of that discussion and he didn't explicitly say anything about the organization name: https://github.com/FossifyX/General-Discussion/issues/2#issuecomment-1837969748 (I'm not sure if that's because he thought the organization and its apps will have matching names).

But... this is a community fork, not a corporation. If enough people want something, it'll happen. Just like the app name prefix poll votes started favoring Pure-Gallery, and Pure-Calendar as soon as Brodie posted his video 4 hours ago saying Pure-* is much better and linked our project (thanks for that!).

With that said, we won't be doing polls for every little detail because we are on a clock. @Hebgbs If you are serious about changing the project name, bring lots of more traffic demanding the name change and it'll probably happen :+1:

**Comment by naveensingh** [comment]
> Throw your hat in the ring and let's see what sticks.

We don't want to spend more time finding better names because:
 - Most of the people responding here are okay with it. Some quite like it and even prefer it over the longer SimpleMobileTools. Hell, we liked FossifyX too.
 - We don't have that kinda time. As you may already know, this is not a commercial project. 
 - We all have subjective experiences and we are all different. There will always be some who do not like the name we choose and I think that's okay.

**Comment by inson1** [comment]
@naveensingh I would move it at least to discussions :)

**Comment by naveensingh** [comment]
@inson1 do you want me to convert this issue into a discussion? or create a separate discussion to change the organization name?

**Comment by inson1** [comment]
@naveensingh convert :) I dont think it should be issue :) @Hebgbs doesnt have enough support for that
But I would let something open, so people dont create new ones and if yes or they ask we could just redirect them there

**Reply by naveensingh** [reply]
done
